tests/neo.UnitTests/SmartContract/UT_JsonSerializer.cs

```diff
@@ -294,7 +294,7 @@ public void Serialize_Array_OfArray()
 
             var json = JsonSerializer.Serialize(entry).ToString();
 
-            Assert.AreEqual(json, "[[true,\"dGVzdDE=\",123],[true,\"dGVzdDI=\",321]]");
+            Assert.AreEqual(json, "[[1,\"dGVzdDE=\",123],[1,\"dGVzdDI=\",321]]");
```

Comment:
Or should we add `Boolean` back?

Reply:
I didn't notice we removed Boolean... but I think that's good, since we have already many stack items. Perhaps `Deserialize('true')` should be typecast into int manually before, so we are more "semantically correct" regarding the Re-Serialization. I don't think it's a big issue.

Reply:
I prefer to reduce the Stack Items, as @igormcoelho said, but at the same time, I think that we should be able to create a complete json too... I don't know which option it's better

Reply:
If you want to keep the boolean type during JSON serialization, you can only add the `Boolean` type back in NeoVM. Otherwise, you should use `Integer`. Either will work for me, but we have to make a choice.

Reply:
@neo-project/core what are your thoughts?
I prefer to return boolean, but we will have the same problems with other types as decimal, or maybe in the future we add different format and we can't depend on this ...

Reply:
Do we support floating-point arithmetic in the future?

Reply:
I mean... maybe xml, and we have a problem with more types, so maybe the solution is not the Boolean stackItem. I think that we should use an array for specify the type of every JObject in the case of json, and make this transparent to the VM. If you want to serialize an object you will need to create this JObject first.

Reply:
Using a JSON to serialize/deserialize another JSON, I think it is too complicated.

Reply:
I personally prefer to restore these basic types, such as boolean.  On the one hand, it can greatly simplify the work at the compiler level. On the other hand, since Erik added the `Slot` feature, our VM is changing from a low-level VM to a IL-level VM. I think this is a very good direction.

Reply:
@neo-project/core Vote?

Reply:
I vote for find a solution without change the stack items